File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocso/OCSACT2GetPatientListCPLInfo.java
package nta.med.data.model.ihis.ocso;

import java.math.BigInteger;

public class OCSACT2GetPatientListCPLInfo {
	private String bunho             ;
	private String suname            ;
	private String inOutGubun      ;
	private String gwa               ;
	private String gwaName          ;
	private String reserYn          ;
	private String doctor            ;
	private String doctorName       ;
	private String kijunDate        ;
	private String emergencyYn      ;
	private String naewonTime       ;
	private BigInteger numProtocol;
	private String suname2           ;
	private Double pkocs1003         ;
	private Double pkout1001         ;
	private String jundalTable      ;
	public OCSACT2GetPatientListCPLInfo(String bunho, String suname, String inOutGubun, String gwa, String gwaName,
			String reserYn, String doctor, String doctorName, String kijunDate, String emergencyYn, String naewonTime,
			BigInteger numProtocol, String suname2, Double pkocs1003, Double pkout1001, String jundalTable) {
		super();
		this.bunho = bunho;
		this.suname = suname;
		this.inOutGubun = inOutGubun;
		this.gwa = gwa;
		this.gwaName = gwaName;
		this.reserYn = reserYn;
		this.doctor = doctor;
		this.doctorName = doctorName;
		this.kijunDate = kijunDate;
		this.emergencyYn = emergencyYn;
		this.naewonTime = naewonTime;
		this.numProtocol = numProtocol;
		this.suname2 = suname2;
		this.pkocs1003 = pkocs1003;
		this.pkout1001 = pkout1001;
		this.jundalTable = jundalTable;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public String getSuname() {
		return suname;
	}
	public void setSuname(String suname) {
		this.suname = suname;
	}
	public String getInOutGubun() {
		return inOutGubun;
	}
	public void setInOutGubun(String inOutGubun) {
		this.inOutGubun = inOutGubun;
	}
	public String getGwa() {
		return gwa;
	}
	public void setGwa(String gwa) {
		this.gwa = gwa;
	}
	public String getGwaName() {
		return gwaName;
	}
	public void setGwaName(String gwaName) {
		this.gwaName = gwaName;
	}
	public String getReserYn() {
		return reserYn;
	}
	public void setReserYn(String reserYn) {
		this.reserYn = reserYn;
	}
	public String getDoctor() {
		return doctor;
	}
	public void setDoctor(String doctor) {
		this.doctor = doctor;
	}
	public String getDoctorName() {
		return doctorName;
	}
	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}
	public String getKijunDate() {
		return kijunDate;
	}
	public void setKijunDate(String kijunDate) {
		this.kijunDate = kijunDate;
	}
	public String getEmergencyYn() {
		return emergencyYn;
	}
	public void setEmergencyYn(String emergencyYn) {
		this.emergencyYn = emergencyYn;
	}
	public String getNaewonTime() {
		return naewonTime;
	}
	public void setNaewonTime(String naewonTime) {
		this.naewonTime = naewonTime;
	}
	public BigInteger getNumProtocol() {
		return numProtocol;
	}
	public void setNumProtocol(BigInteger numProtocol) {
		this.numProtocol = numProtocol;
	}
	public String getSuname2() {
		return suname2;
	}
	public void setSuname2(String suname2) {
		this.suname2 = suname2;
	}
	public Double getPkocs1003() {
		return pkocs1003;
	}
	public void setPkocs1003(Double pkocs1003) {
		this.pkocs1003 = pkocs1003;
	}
	public Double getPkout1001() {
		return pkout1001;
	}
	public void setPkout1001(Double pkout1001) {
		this.pkout1001 = pkout1001;
	}
	public String getJundalTable() {
		return jundalTable;
	}
	public void setJundalTable(String jundalTable) {
		this.jundalTable = jundalTable;
	}
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocso/OCSACT2GetPatientListCPLInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocso
{
    public class OCSACT2GetPatientListCPLInfo
    {
        private String _bunho;
        private String _suname;
        private String _inOutGubun;
        private String _gwa;
        private String _gwaName;
        private String _reserYn;
        private String _doctor;
        private String _doctorName;
        private String _kijunDate;
        private String _emergencyYn;
        private String _naewonTime;
        private String _trialYn;
        private String _suname2;
        private String _pkocs1003;
        private String _pkout1001;
        private String _jundalTable;

        public String Bunho
        {
            get { return this._bunho; }
            set { this._bunho = value; }
        }

        public String Suname
        {
            get { return this._suname; }
            set { this._suname = value; }
        }

        public String InOutGubun
        {
            get { return this._inOutGubun; }
            set { this._inOutGubun = value; }
        }

        public String Gwa
        {
            get { return this._gwa; }
            set { this._gwa = value; }
        }

        public String GwaName
        {
            get { return this._gwaName; }
            set { this._gwaName = value; }
        }

        public String ReserYn
        {
            get { return this._reserYn; }
            set { this._reserYn = value; }
        }

        public String Doctor
        {
            get { return this._doctor; }
            set { this._doctor = value; }
        }

        public String DoctorName
        {
            get { return this._doctorName; }
            set { this._doctorName = value; }
        }

        public String KijunDate
        {
            get { return this._kijunDate; }
            set { this._kijunDate = value; }
        }

        public String EmergencyYn
        {
            get { return this._emergencyYn; }
            set { this._emergencyYn = value; }
        }

        public String NaewonTime
        {
            get { return this._naewonTime; }
            set { this._naewonTime = value; }
        }

        public String TrialYn
        {
            get { return this._trialYn; }
            set { this._trialYn = value; }
        }

        public String Suname2
        {
            get { return this._suname2; }
            set { this._suname2 = value; }
        }

        public String Pkocs1003
        {
            get { return this._pkocs1003; }
            set { this._pkocs1003 = value; }
        }

        public String Pkout1001
        {
            get { return this._pkout1001; }
            set { this._pkout1001 = value; }
        }

        public String JundalTable
        {
            get { return this._jundalTable; }
            set { this._jundalTable = value; }
        }

        public OCSACT2GetPatientListCPLInfo() { }

        public OCSACT2GetPatientListCPLInfo(String bunho, String suname, String inOutGubun, String gwa, String gwaName, String reserYn, String doctor, String doctorName, String kijunDate, String emergencyYn, String naewonTime, String trialYn, String suname2, String pkocs1003, String pkout1001, String jundalTable)
        {
            this._bunho = bunho;
            this._suname = suname;
            this._inOutGubun = inOutGubun;
            this._gwa = gwa;
            this._gwaName = gwaName;
            this._reserYn = reserYn;
            this._doctor = doctor;
            this._doctorName = doctorName;
            this._kijunDate = kijunDate;
            this._emergencyYn = emergencyYn;
            this._naewonTime = naewonTime;
            this._trialYn = trialYn;
            this._suname2 = suname2;
            this._pkocs1003 = pkocs1003;
            this._pkout1001 = pkout1001;
            this._jundalTable = jundalTable;
        }

    }
}